cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -197,3 +161,27 @@ the nodes of any clusters where {CNVProductName} is deployed. The nodes might no
 drain if there are virtual machines running on top of them.
 The current solution is to put nodes into maintenance.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1707427[*BZ#1707427*])
+
+* When running `virtctl image-upload` to upload large VM disk images in `qcow2`
+format, an end-of-file (EOF) error may be reported after the data is
+transmitted, even though the upload is either progressing normally or completed.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1754920[*BZ#1754920*])
++
+Run the following command to check the status of an upload on a given PVC:
++
+----
+$ oc describe pvc <pvc-name> | grep cdi.kubevirt.io/storage.pod.phase
+----
+
+* When attempting to create a virtual machine on hardware with a Haswell CPU,
+the virtual machine creation fails and `libvert` declares
+`<model usable='yes'>Haswell</model>`. This is a change in behavior from
+previous verions of {CNVProductName}, where `libvirt` declared
+`<model usable='no'>Haswell</model>`.
```

Comment:
I've proposed this text in the relevant bug, pasting it here (adjusted to the recent discussion):

```
"""
When attempting to create and kaunch a virtual machine using a Haswell CPU (model: Haswell), then the launch of the virtual machine fails. This is a bug due to incorrectly labeled nodes and a change in behaviour from previous versions of container-native virtualization, where VMs could be successfully launched on Haswell hosts.

As a workaround, select a different CPU model, if possible.
```

Reply:
Updated.